src/site/content/en/blog/optimize-lcp/index.md

```diff
@@ -38,29 +38,90 @@ To provide a good user experience, **sites should strive to have an LCP of 2.5 s
+
+Once a page starts rendering there may be an initial paint (for example, the background color is painted), followed by some content showing up (for example, the site header). The appearnce of the initial conent is measured by FCP. The delta between FCP and other metrics can be very telling.
+
+A large delta between TTFB and FCP could indicate that the browser needs to download a lot of render-blocking assets. It can also be a sign it must complete a lot of work to render any meaningful content—a classic sign of a Client Side Rendered site.
+
+A large delta between FCP and LCP indicates that the LCP resource is either not immediately available for the browser to prioritise (for example, text or images that are managed by JavaScript rather than being available in the initial HTML) or that the browser is completing other work before it can display the LCP content.
+
+### Using PageSpeed Insights Lighthouse data
+
+The Lighthouse section of PageSpeed Insights offers some guidance to improving LCP, but first you should check if the LCP given is broadly inline with real user data provided by CrUX.
+
+If Lighthouse is showing no LCP issue, but the CrUX data is, then any Lighthouse suggestions may not be relevant. The opposite is also true—if Lighthouse is showing a really poor LCP time, but CrUX data is showing your users mostly have a good LCP, then it may not be necessary to further improve LCP. Just be sure to check that the CrUX data is for this page and not for the full origin as detailed above.
+
+If both sources of data are showing an LCP that should be improved than the Lighthouse section can provide valuable guidance on ways to improve LCP. Use the LCP filter to only show audits relevant to LCP:
+
+{% Img src="image/W3z1f5ZkBJSgL1V1IfloTIctbIF3/himAvC1GMr7K0kcbvG4F.png", alt="Lighthouse LCP Opportunities and Diagnostics", width="800", height="498" %}
+
+As well as the Opportunities to improve, there is Diagnostic information that may provide more information to help diagnose the issue. The **Largest Contentful Paint elment** diagnostic shows a useful breakdown of the various timings that made up the LCP:
+
+{% Img src="image/W3z1f5ZkBJSgL1V1IfloTIctbIF3/KhIuP2CNToNeGa33w48K.png", alt="Lighthouse LCP phases", width="800", height="474" %}
+
+We will delve into these sub-parts next.
+
 ## LCP breakdown
 
+Optimizing for LCP can be a more complex task when PageSpeed Insights does not give you the answer on how to improve this metric. With complex tasks it's generally better to break them down into smaller, more manageable tasks and address each separately. This guide will present a methodology for how to break down LCP into its most critical sub-parts and then present specific recommendations and best practices for how to optimize each part.
+
+{% Aside %}
+  For a visual overview of the context presented in this section, see [A Deep Dive into Optimizing LCP](https://youtu.be/fWoI9DXmpdk) from Google I/O '22:
+  {% YouTube "fWoI9DXmpdk" %}
```

Comment:
Hm, do we want to put the YouTube embed in an aside? It might look strange, and might be better to pull this out into body copy so the embed flows better with the content.

Reply:
I didn't change this. Just git has got confused as I added another Aside. So this is how it appears now and think it looks fine: https://web.dev/optimize-lcp/#:~:text=For%20a%20visual%20overview%20of%20the%20context%20presented%20in%20this%20guide

<img width="993" alt="image" src="https://github.com/GoogleChrome/web.dev/assets/10931297/bb9fc264-9214-4848-b24c-458a53e7c978">

Reply:
Ah, no worries then. Looks better than I imagined it in my head 😅